pkg/scheduler/api/resource_info/gpu_resource_requirment.go

```diff
@@ -158,6 +158,9 @@ func (g *GpuResourceRequirement) GPUs() float64 {
 }
 
 func (g *GpuResourceRequirement) GetNumOfGpuDevices() int64 {
+	if g.portion == 0 && g.gpuMemory == 0 {
+		return 0
+	}
```

Comment:
I think that this case should be addressed in the new NewGpuResourceRequirementWithGpus as well.

Reply:
:+1: